docker-compose.yml

```diff
@@ -59,3 +60,67 @@ services:
       interval: 10s
       timeout: 120s
       retries: 10
+
+  elasticsearch:
+    build:
+      context: elasticsearch/
+      args:
+        ELK_VERSION: $ELK_VERSION
+    volumes:
+      - type: bind
+        source: ./elasticsearch/config/elasticsearch.yml
+        target: /usr/share/elasticsearch/config/elasticsearch.yml
+        read_only: true
```

Comment:
is the data persisted elswere?

Reply:
This data will be in data folder in the container. We can delete those data through kibana UI or by elastic API.